axelor-production/src/main/java/com/axelor/apps/production/service/productionorder/ProductionOrderServiceImpl.java

```diff
@@ -139,7 +142,14 @@ public ProductionOrder addManufOrder(
       if (saleOrder != null) {
         manufOrder.addSaleOrderSetItem(saleOrder);
         manufOrder.setClientPartner(saleOrder.getClientPartner());
-        manufOrder.setMoCommentFromSaleOrder(saleOrder.getProductionNote());
+        if (Strings.isNullOrEmpty(saleOrder.getProductionNote())) {
+          manufOrder.setMoCommentFromSaleOrder(moCommentFromSaleOrderLine);
+        } else {
+          manufOrder.setMoCommentFromSaleOrder(
+              saleOrder
+                  .getProductionNote()
+                  .concat(System.lineSeparator().concat(moCommentFromSaleOrderLine)));
```

Comment:
Please check if `moCommentFromSaleOrderLine` is null to avoid a NullPointerException.